doc/model/fluid.rst

```diff
@@ -0,0 +1,189 @@
+=====
+Fluid
+=====
+
+.. note::
+
+   This model document has been updated to be |mirgecom| specific, and to include
+   multi-component mixtures with chemical reactions.
+
+.. important::
+
+   |mirgecom| main repo branch currently provides fluid simulation operators and
+   utilities for inviscid flow (i.e. the :ref:`Euler equations<euler-eqns>`).
+   Viscous flow simulation capabilities are forthcoming.
+
+
+.. _ns-eqns:
+
+|mirgecom| provides capabilities for solving the compressible Navier-Stokes equations for
+viscous flows and the :ref:`Euler equations<euler-eqns>` equations for inviscid flows of
+reactive fluid mixtures. |mirgecom| supports reactive fluid mixtures with a number of
+mixture species $N_s$ on unstructured meshes of 1, 2, and 3-dimensional domains $\Omega$,
+and :ref:`discretizes the equations in a Discontinuous Galerkin setting<disc-strat>`.
+
+The formulation presented here follows [Ihme_2014]_ and [Cook_2009]_. Unless otherwise
+noted, Einstein summation convention is used throughout the following sections.  The
+governing equations, written in conservative form, are summarized as follows:
+
+$$
+\partial_{t}{\rho} + \partial_{j}{\rho v_j} &= {S}_\rho \\
+\partial_{t}(\rho{E}) + \partial_j\left(\left\{\rho E + p\right\}v_j + q_j -
+\tau_{jk}v_k\right) &= {S}_{\rho E} \\
+\partial_{t}({\rho}{v_i}) + \partial_j\left(\rho v_i v_j + p\delta_{ij} -
+\tau_{ij}\right) &= {S}_{\rho v_i} \\
+\partial_{t}(\rho{Y}_{\alpha}) + \partial_j\left(\rho{Y}_{\alpha}v_j +
+(\mathbf{J}_{\alpha})_j\right) &= {S}_{\alpha},
+$$
+
+with fluid density $\rho$, velocity components $v_i$, momentum density components
+$(\rho v_i$), total energy $(\rho E)$, and vector of species mass fractions
+${Y}_{\alpha}$. The :ref:`thermodynamic pressure<eos-and-matprop>` of the fluid is $p$.
+${\tau_{ij}}$ are the components of the
+:ref:`viscous stress tensor<viscous-stress-tensor>`, $q_i$ are the components of the total
+:ref:`heat flux<heat-flux>` vector, and the components of the species
+:ref:`diffusive flux<diffusive-flux>` vector are $(\mathbf{J}_{\alpha})_i$. Mixtures have
+$N_s$ components with $1 \le \alpha \le N_s$.
+
+The equations can be recast in a more compact form:
+
+$$
+\partial_t{\mathbf{Q}} + \partial_j{\mathbf{F}^{I}_j} = \partial_j{\mathbf{F}^{V}_j} +
+\mathbf{S},
```

Comment:
```suggestion
\partial_t{\mathbf{Q}} + \partial_j{\mathbf{F}^{I}(\mathbf{Q})_j} =
\partial_j{\mathbf{F}^{V}(\mathbf{Q}, \nabla\mathbf{Q})_j} + \mathbf{S},
```
This is a matter of personal taste, but I think explicitly stating the dependence on the state (especially reinforcing that the viscous flux depends on the gradient of the state) is helpful